src/content/self-hosted/install/releases.mdx

```diff
@@ -5,6 +5,47 @@ sidebar_label: Release Notes
 id: releases
 ---
 
+## 1.5.0
+14 February, 2023
+
+### Breaking Change
+
+- Create dedicated registry serviceAccounts by default
+
+### Features
+
+- Added antiAffinity (configuration)[/docs/self-hosted/administration/configuration/#userpodaffinity] for PODs with label `dev.okteto.com/affinity`
+- Add external resource icons
+- Build credential helpers
+- Remove load time on GitHub redeploy dialog
+
+### Bug Fix
+
+- When injecting host alises to go through internal network to buildkit, registry and api, don't duplicate entries for same IP. Instead, pass the list of hostnames for the IP
+- Delete button is now disabled in the personal namespace
+- Restore k3s support
+- Fix faulty error case on deployment with external resource
+- Fix faulty progressing state on deployment with external resource
+- Disable CTA while destroying all
+- Catch all non `/error/{code}` requests
+- Make env var values editable on deploy/redeploy
+- Keep ingress class on wake if ingress class is different than "sleeping"
+- Fix service unavailable when destroy all
+- Fix divert snippet configuration
+- Filter repositories results by search term
+- Fix pointer cursor in staged log lines
```

Comment:
What does this fix?

Reply:
@rlamana https://github.com/okteto/app/pull/5537 can you give more contexT?

Reply:
Jumping in.
Resource log lines were inheriting a `cursor` property of type `pointer` from the parent container, which was giving the idea that log lines were actionable, when they weren't.
This PR reverts to use default cursor.

Reply:
Maybe it should say `Fixed wrong cursor when hovering logs lines, they can now be selected as normal text`